CMP1903_A1_2324/Die.cs

```diff
@@ -13,11 +13,12 @@ internal class Die
          * and one method that rolls the die, returns and integer and takes no parameters.
          */
 
-        //Property
-
-
-        //Method
-
-
+        private static Random random = new Random();
+        public int diceValue { get; set; } //stores values after rolling dice
+        public int Roll() //roll dice
+        {
+            diceValue = random.Next(1, 7); //generates a random number between 1 and 6
+            return diceValue;
+        }
     }
 }
```

Comment:
Die class has met the assignment requirements, it simulates a die that is assigned a random value between 1 and 6. Easy to follow and read, meets requirements of C#. Only thing that could be improved is using comments to make it easier to follow.